src/APIM_ARMTemplate/apimtemplate.test/Creator/TemplateCreatorTests/MasterTemplateCreatorTests.cs

```diff
@@ -24,7 +24,7 @@ public void ShouldCreateCorrectNumberOfDeploymentResources()
             int count = 5;
 
             // act
-            Template masterTemplate = masterTemplateCreator.CreateLinkedMasterTemplate(apiVersionSetsTemplate, productsTemplate, loggersTemplate, null, null, apiInfoList, creatorFileNames, creatorConfig.apimServiceName, fileNameGenerator);
+            Template masterTemplate = masterTemplateCreator.CreateLinkedMasterTemplate(apiVersionSetsTemplate, productsTemplate, loggersTemplate, null, null, apiInfoList, creatorFileNames, creatorConfig.apimServiceName, fileNameGenerator, false, null);
```

Comment:
Lets try use named arguments. Improves readability

Reply:
done. howerver this is a test class code so not so important. I think the whole method have too many parameters already and some clean up will be required sooner or later.